README.rst

```diff
@@ -41,7 +41,22 @@ MNE-BIDS
 
 This is a repository for creating
 `BIDS <http://bids.neuroimaging.io/>`_\ -compatible datasets with
-`MNE <https://mne.tools/stable/index.html>`_.
+`MNE-Python <https://mne.tools/stable/index.html>`_.
+
+BIDS (Brain Imaging Data Structure) is a standard to organize data
+according to a set of rules that describe ...
+
+- how to name your files
+- where to place your files within a directory structure
+- what additional metadata to encode, and how to encode it
```

Comment:
```suggestion
- what additional metadata to store, and how to store it in sidecar json and tsv files
```

I'm not sure what you mean by "encode", changing it to something simpler like store might be better.